### ISSUE TITLE
Cannot Open the App

### ISSUE THREAD

**Issue by ashcoft** [rotdrop/nextcloud-roundcube#9]
here is the screenshot
![image](https://user-images.githubusercontent.com/1115854/218945653-5cd55629-add0-45e2-a834-ba1875598052.png)

**Comment by rotdrop** [comment]
Can you please give some more information about your server installation (RC version, NC version is probably 25, RC behind proxy or not, server OS, kind of web-browser)? Of course, generating a cryptic error message in itself can be considered a bug ;)

Some other thoughts:

- anything more in the log-files at the time this is happening (either the Nextcloud log or your web-server log-files)?
- is the Roundcube installation working if you call it directly from you browser?

In particular the first point -- what can be found in the log files? -- would be particularly helpful.

**Comment by alauzon** [comment]
I have the same problem.
sudo -u www-data php --define apc.enable_cli=1  /var/www/nextcloud/occ support:report
## Server configuration detail

**Operating system:** Linux 5.15.39-3-pve #2 SMP PVE 5.15.39-3 (Wed, 27 Jul 2022 13:45:39 +0200) x86_64

**Webserver:** Unknown (cli)

**Database:** mysql 10.3.38

**PHP version:** 8.1.5

Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, json, pcntl, Reflection, SPL, session, standard, sodium, mysqlnd, PDO, xml, apcu, bcmath, bz2, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, imagick, intl, exif, mysqli, pdo_mysql, pdo_sqlite, Phar, posix, readline, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xmlreader, xmlwriter, xsl, zip, Zend OPcache

**Nextcloud version:** 25.0.3 - 25.0.3.2

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

[]
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.17.0
 - admin_audit: 1.15.0
 - apporder: 0.15.0
 - audioplayer: 3.3.1
 - bbb: 2.4.0
 - bruteforcesettings: 2.5.0
 - calendar: 4.2.4
 - circles: 25.0.0
 - cloud_federation_api: 1.8.0
 - comments: 1.15.0
 - contacts: 5.1.0
 - contactsinteraction: 1.6.0
 - dashboard: 7.5.0
 - dav: 1.24.0
 - deck: 1.8.3
 - drawio: 2.1.0
 - federatedfilesharing: 1.15.0
 - federation: 1.15.0
 - files: 1.20.1
 - files_mindmap: 0.0.27
 - files_pdfviewer: 2.6.0
 - files_rightclick: 1.4.0
 - files_sharing: 1.17.0
 - files_trashbin: 1.15.0
 - files_versions: 1.18.0
 - firstrunwizard: 2.14.0
 - forms: 3.1.0
 - group_everyone: 0.1.11
 - groupfolders: 13.1.1
 - impersonate: 1.12.0
 - logreader: 2.10.0
 - lookup_server_connector: 1.13.0
 - mail: 2.2.4
 - maps: 0.2.4
 - nextcloud_announcements: 1.14.0
 - notes: 4.6.0
 - notifications: 2.13.1
 - oauth2: 1.13.0
 - password_policy: 1.15.0
 - photos: 2.0.1
 - privacy: 1.9.0
 - provisioning_api: 1.15.0
 - recognize: 3.5.0
 - recommendations: 1.4.0
 - registration: 2.1.0
 - related_resources: 1.0.3
 - richdocuments: 7.1.1
 - serverinfo: 1.15.0
 - settings: 1.7.0
 - sharebymail: 1.15.0
 - snappymail: 2.26.4
 - spreed: 15.0.4
 - support: 1.8.0
 - survey_client: 1.13.0
 - suspicious_login: 4.3.0
 - systemtags: 1.15.0
 - tasks: 0.14.5
 - text: 3.6.0
 - theming: 2.0.1
 - twofactor_backupcodes: 1.14.0
 - twofactor_totp: 7.0.0
 - updatenotification: 1.15.0
 - user_status: 1.5.0
 - viewer: 1.9.0
 - weather_status: 1.5.0
 - workflowengine: 2.7.0
Disabled:
 - bookmarks_fulltextsearch: 1.2.0
 - encryption
 - files_external
 - files_fulltextsearch: 25.0.0
 - files_fulltextsearch_tesseract: 25.0.0
 - fulltextsearch: 25.0.0
 - fulltextsearch_elasticsearch: 25.0.0
 - mail_roundcube: 1.0.1
 - richdocumentscode: 22.5.802
 - sociallogin: 5.4.1
 - socialsharing_email: 2.5.0
 - socialsharing_facebook: 2.5.0
 - socialsharing_twitter: 2.5.0
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "cloud.serveursdupeuple.net",
        "198.50.187.237",
        "193.46.255.49",
        "198.50.187.237:80"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "25.0.3.2",
    "overwrite.cli.url": "https:\/\/cloud.serveursdupeuple.net",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "mysql.utf8mb4": true,
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "maintenance": false,
    "htaccess.RewriteBase": "\/",
    "theme": "",
    "loglevel": 1,
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpmode": "smtp",
    "mail_sendmailmode": "smtp",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "defaultapp": "quicknotes,dashboard",
    "default_language": "fr",
    "default_locale": "fr_CA",
    "default_phone_region": "CA",
    "has_rebuilt_cache": true,
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
    "forwarded_for_headers": [
        "HTTP_X_FORWARDED_FOR"
    ],
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpsecure": "tls",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "app_install_overwrite": [
        "apporder",
        "deck",
        "bbb",
        "richdocumentscode",
        "drawio",
        "fulltextsearch",
        "bookmarks_fulltextsearch",
        "fulltextsearch_elasticsearch",
        "files_fulltextsearch",
        "files_fulltextsearch_tesseract",
        "defaultgroup",
        "mindmap_app"
    ],
    "updater.release.channel": "stable",
    "log_type": "file"
}
```
</details>

**Cron Configuration:** Array
(
    [backgroundjobs_mode] => cron
    [lastcron] => 1677942914
)


**External storages:** files_external is disabled

**Encryption:** no

**User-backends:** 
 * OC\User\Database


**Talk configuration:** 

STUN servers
 * sturn.serveursdupeuple.net:2443

TURN servers
 * turn:sturn.serveursdupeuple.net:2443 - tcp

Signaling servers (mode: default):
 * no custom server configured

Error log:
{"reqId":"J0GlJCY0xEX3kbgIoInI","level":3,"time":"2023-03-04T15:03:05+00:00","remoteAddr":"66.110.158.64","user":"Alain Lauzon","app":"PHP","method":"GET","url":"/apps/mail_roundcube/","message":"Trying to access array offset on value of type null at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#92","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36","version":"25.0.3.2","exception":{"Exception":"Error","Message":"Trying to access array offset on value of type null at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#92","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php","line":92,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"function":"__construct","class":"OCA\\RoundCube\\Service\\AuthRoundCube","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"newInstanceArgs","class":"ReflectionClass","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":89,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"function":"OC\\AppFramework\\Utility\\{closure}","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"array_map"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":159,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/nextcloud/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"J0GlJCY0xEX3kbgIoInI","level":3,"time":"2023-03-04T15:03:05+00:00","remoteAddr":"66.110.158.64","user":"Alain Lauzon","app":"PHP","method":"GET","url":"/apps/mail_roundcube/","message":"Undefined array key \"scheme\" at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#100","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36","version":"25.0.3.2","exception":{"Exception":"Error","Message":"Undefined array key \"scheme\" at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#100","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php","line":100,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"function":"__construct","class":"OCA\\RoundCube\\Service\\AuthRoundCube","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"newInstanceArgs","class":"ReflectionClass","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":89,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"function":"OC\\AppFramework\\Utility\\{closure}","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"array_map"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":159,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/nextcloud/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"J0GlJCY0xEX3kbgIoInI","level":3,"time":"2023-03-04T15:03:05+00:00","remoteAddr":"66.110.158.64","user":"Alain Lauzon","app":"PHP","method":"GET","url":"/apps/mail_roundcube/","message":"Undefined array key \"host\" at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#101","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36","version":"25.0.3.2","exception":{"Exception":"Error","Message":"Undefined array key \"host\" at /var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#101","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php","line":101,"function":"onError","class":"OC\\Log\\ErrorHandler","type":"::"},{"function":"__construct","class":"OCA\\RoundCube\\Service\\AuthRoundCube","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"newInstanceArgs","class":"ReflectionClass","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":89,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"function":"OC\\AppFramework\\Utility\\{closure}","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->","args":["*** sensitive parameters replaced ***"]},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":108,"function":"array_map"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":116,"function":"buildClass","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php","line":133,"function":"resolve","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":465,"function":"query","class":"OC\\AppFramework\\Utility\\SimpleContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php","line":437,"function":"queryNoFallback","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":159,"function":"query","class":"OC\\AppFramework\\DependencyInjection\\DIContainer","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/nextcloud/lib/private/Log/ErrorHandler.php","Line":92,"CustomMessage":"--"}}
{"reqId":"J0GlJCY0xEX3kbgIoInI","level":3,"time":"2023-03-04T15:03:05+00:00","remoteAddr":"66.110.158.64","user":"Alain Lauzon","app":"mail_roundcube","method":"GET","url":"/apps/mail_roundcube/","message":"/var/www/nextcloud/apps/mail_roundcube/lib/Service/Config.php:326: OCA\\RoundCube\\Service\\Config::emailCredentials(): Unable to obtain email credentials for user Alain Lauzon","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36","version":"25.0.3.2","data":{"app":"mail_roundcube"}}
{"reqId":"J0GlJCY0xEX3kbgIoInI","level":3,"time":"2023-03-04T15:03:05+00:00","remoteAddr":"66.110.158.64","user":"Alain Lauzon","app":"index","method":"GET","url":"/apps/mail_roundcube/","message":"OCA\\RoundCube\\Service\\AuthRoundCube::externalURL(): Return value must be of type string, null returned in file '/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php' line 144","userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36","version":"25.0.3.2","exception":{"Exception":"Exception","Message":"OCA\\RoundCube\\Service\\AuthRoundCube::externalURL(): Return value must be of type string, null returned in file '/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php' line 144","Code":0,"Trace":[{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","Line":165,"Previous":{"Exception":"TypeError","Message":"OCA\\RoundCube\\Service\\AuthRoundCube::externalURL(): Return value must be of type string, null returned","Code":0,"Trace":[{"file":"/var/www/nextcloud/apps/mail_roundcube/lib/Controller/PageController.php","line":124,"function":"externalURL","class":"OCA\\RoundCube\\Service\\AuthRoundCube","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":225,"function":"index","class":"OCA\\RoundCube\\Controller\\PageController","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php","line":133,"function":"executeController","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/AppFramework/App.php","line":172,"function":"dispatch","class":"OC\\AppFramework\\Http\\Dispatcher","type":"->"},{"file":"/var/www/nextcloud/lib/private/Route/Router.php","line":298,"function":"main","class":"OC\\AppFramework\\App","type":"::"},{"file":"/var/www/nextcloud/lib/base.php","line":1047,"function":"match","class":"OC\\Route\\Router","type":"->"},{"file":"/var/www/nextcloud/index.php","line":36,"function":"handleRequest","class":"OC","type":"::"}],"File":"/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php","Line":144},"CustomMessage":"--"}}

No proxy, 
cat /proc/version
Linux version 5.15.39-3-pve (build@proxmox) (gcc (Debian 10.2.1-6) 10.2.1 20210110, GNU ld (GNU Binutils for Debian) 2.35.2) #2 SMP PVE 5.15.39-3 (Wed, 27 Jul 2022 13:45:39 +0200)
Browser: Chrome

**Comment by rotdrop** [comment]
Hi there, this
```
/var/www/nextcloud/apps/mail_roundcube/lib/Service/AuthRoundCube.php#92
```
would perhaps mean that you did not configure the Roundcube location? Around line 92 we have

https://github.com/rotdrop/nextcloud-roundcube/blob/5966d36180a7b90e811d58ba10b6d6c693ef05d8/lib/Service/AuthRoundCube.php#L91-L96

Can you please verify that in the admin-settings the  setting for the RC location (still) filled out?

If so, then yes: the error message really could be improved ...

**Comment by HLFH** [comment]
Yes, EXTERNAL_LOCATION is not configured in that case. I can reproduce this issue. Maybe the displayed error message could be improved and inform about the real issue.

**Comment by rotdrop** [comment]
This should be fixed with a60fee1. Closing this issue now.
